### ISSUE TITLE
Not working with dart command line application

### ISSUE THREAD

**Issue by desmondlzy** [vitusortner/floor#578]
Hi! 

TL;DR: `import 'package:floor/floor.dart';` doesn't work in a dart command line app because floor depends on flutter.

So here I'm tryna build an image managing desktop app that uses floor for persistent storage and the app comes with two interfaces: a gui and a cli. That says, In addition to the flutter desktop app, my project has another build target: a dart command line app. 

Floor works great for the flutter app. But when I attempt to run from the cli entry point, it gives the error `Not found: 'dart:ui'`. I googled a bit and found this [SO thread](https://stackoverflow.com/questions/52483773/dartui1-error-not-found-dartui-flutter-dart) and this [issue](https://github.com/flutter/flutter/issues/72452). I also tried deleting floor-related imports and then it worked. Seems that it simply doesn't work importing floor (or flutter package) from a dart cli app.

I understand this could be hard to resolve. I noticed floor uses sqflite which also heavily depends on flutter, so......

Do you have any plan to support it in the future? or any workarounds? It's bit unnatural to me a database orm needs to depend on a UI framework... Floor is by far the best database orm I find on dart in terms of API design, so I'm just tryna..

**Comment by jhionan** [comment]
the import is the problem, you should not import floor/floor.dart

you should `import 'package:floor/src/your_file.dart'`

need to import directly from `src` folder not the file that exports the others.

**Comment by desmondlzy** [comment]
Hey thanks it's working!

**Comment by ollydixon** [comment]
@desmondlzy but that won't compile since it can't find it. 

<img width="855" alt="Screenshot 2021-07-21 at 18 56 01" src="https://user-images.githubusercontent.com/6940726/126528590-3adcdb87-c5f3-489b-94d8-a582d2ec995e.png">

**Comment by alimcomp** [comment]
Hi and thank you for that great library 
I'v got the same issue when im trying to use floor in cmd application

**Comment by dkaera** [comment]
> @jhionan but that won't compile since it can't find it.
> 
> <img alt="Screenshot 2021-07-21 at 18 56 01" width="855" src="https://user-images.githubusercontent.com/6940726/126528590-3adcdb87-c5f3-489b-94d8-a582d2ec995e.png">

I'm not sure if the path in the screenshot is part of Floor.
